client/js/jsonedit.js

```diff
@@ -477,6 +478,19 @@ function jeAddCommands() {
   jeAddNumberCommand('opposite value', '0', x=>-x);
 }
 
+function displayComputeOps() {
+  const keyword = document.getElementById('var_search').value;
```

Comment:
`$('#var_search')` would be shorter (but also minimally slower). Same for the other uses.